apps/scan/backend/src/index.ts

```diff
@@ -70,6 +70,7 @@ async function main(): Promise<number> {
     precinctScannerStateMachine,
     workspace,
     usbDrive,
+    logger,
```

Comment:
I wasn't quite sure why logger wasn't being passed here, which the effect would be that there are two different logger instances when the app is running. I'm not sure that's a problem, but thought it made sense to pass here.